diff -up at-3.1.20/atd.8.in.document-n at-3.1.20/atd.8.in
--- at-3.1.20/atd.8.in.document-n	2015-08-22 00:09:22.000000000 +0200
+++ at-3.1.20/atd.8.in	2017-09-14 14:17:04.922086349 +0200
@@ -9,6 +9,7 @@ atd \- run jobs queued for later executi
 .IR batch_interval ]
 .RB [ -d ]
 .RB [ -f ]
+.RB [ -n ]
 .RB [ -s ]
 .SH DESCRIPTION
 .B atd
@@ -40,6 +41,10 @@ Run
 .BR atd
 in the foreground.
 .TP 8
+.B -n
+Append the hostname of the system to the subject of the e-mails sent by
+.BR atd .
+.TP 8
 .B -s
 Process the at/batch queue only once.
 This is primarily of use for compatibility with old versions of
